patches/chrome-test-BUILD.gn.patch

```diff
@@ -10,3 +10,12 @@ index 5f92e3b598fa3ad8767d20db975cb29329577613..266065286314ff0e07e0a6358e3dc5f0
    data_deps = []
  
    if (enable_pdf) {
+@@ -4968,7 +4969,7 @@ config("disable_thinlto_cache_flags") {
+ }
+ 
+ config("pdb_larger_than_4gb") {
+-  if (is_win && symbol_level == 2 && use_thin_lto) {
```

Comment:
why do we need different conditions here than upstream?

Reply:
also if the only purpose of this change is to use this config in test/BUILD.gn then we should just create our own config for it instead of patching this one

Reply:
@cdesouza-chromium ☝️ was this needed because building of the Chromium tests was failing on CI?

Reply:
Fixed e65909ae8ddb5709ec77b60aeba43594fb4c70dd, according to the suggestion in DMs. I'm also removing the config in our targets because as I remember this was only happening in the upstream tests.